aQute.libg/src/aQute/lib/io/IO.java

```diff
@@ -186,6 +187,19 @@ public static OutputStream copy(InputStream in, OutputStream out) throws IOExcep
 		}
 	}
 
+	public static OutputStream copy(InputStream in, OutputStream out, int limit) throws IOException {
```

Comment:
We already have `LimitedInputStream` for this purpose. So we don't need this method which duplicates the behavior. Just do

```java
IO.copy(new LimitedInputStream(in, limit), out);
```